app/models/promise.js

```diff
@@ -1,24 +1,22 @@
 import Ember from "ember";
 import escapeRegExp from "ember-inspector/utils/escape-reg-exp";
+import computed from 'ember-new-computed';
 var typeOf = Ember.typeOf;
-var computed = Ember.computed;
-var or = computed.or;
-var equal = computed.equal;
-var not = computed.not;
+var or = Ember.computed.or;
+var equal = Ember.computed.equal;
+var not = Ember.computed.not;
 
 var dateComputed = function() {
-  return Ember.computed(
+  return computed({ set:
```

Comment:
Can the `set` be on the same line as its function?  Same as https://github.com/rwjblue/ember-new-computed#usage